pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -659,6 +662,9 @@ func TestGenericScheduler(t *testing.T) {
 			if test.expectedHosts != nil && !test.expectedHosts.Has(result.SuggestedHost) {
 				t.Errorf("Expected: %s, got: %s", test.expectedHosts, result.SuggestedHost)
 			}
+			if test.wErr == nil && len(test.nodes) != result.EvaluatedNodes {
+				t.Errorf("Expected EvaluatedNodes: %d, got: %d", len(test.nodes), result.EvaluatedNodes)
```

Comment:
Nit: it'll be better to print the test name in the log.

Reply:
Test name is already printed like below.
```
--- FAIL: TestGenericScheduler (0.00s)
    --- FAIL: TestGenericScheduler/test_with_partial_failed_filter_plugin (0.00s)
        generic_scheduler_test.go:666: Expected EvaluatedNodes: 2, got: 1
```